package.json

```diff
@@ -83,7 +83,6 @@
     "@griffel/jest-serializer": "1.1.16",
     "@griffel/react": "^1.5.14",
     "@griffel/shadow-dom": "0.1.5",
-    "@griffel/webpack-extraction-plugin": "0.3.18",
```

Comment:
These changes roll back additions [here](https://github.com/microsoft/fluentui/pull/24279). I've not rolled back all changes as some are now used elsewhere.